qbraid/transpiler/tests/coverage/test_from_cirq.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import pytest
+import string
+
+import qbraid
+import cirq
+
+#############
+### UTILS ###
+#############
+
+def generate_params(varnames):
+    params = {}
+    rot_args = ['rads']
+    for ra in rot_args:
+        if ra in varnames:
+            params[ra] = np.random.rand() * 2 * np.pi
+            
+    if 'exponent' in varnames:
+        params['exponent'] = np.random.rand() * 10
+        
+    if 'num_qubits' in varnames:
+        params['num_qubits'] = np.random.randint(1, 7)
+        
+    return params
+
+def get_cirq_gates():
+    qubits = cirq.LineQubit.range(7)
+
+    cirq_gates = {attr: None for attr in dir(cirq.ops.common_gates) if attr[0] in string.ascii_uppercase if  (isinstance(getattr(cirq.ops.common_gates, attr), cirq.value.abc_alt.ABCMetaImplementAnyOneOf) or isinstance(getattr(cirq.ops.common_gates, attr), cirq.Gate))}
```

Comment:
This a great programmatic way to get the cirq gates! The `cirq.ops.common_gates` are a great start. To make sure we are getting good, representative coverage, could you also add support for some of the other widely used gate modules such as `cirq.ops.pauli_gates` and `cirq.ops.parity_gates`, as well as some other widely used individual gates such as `cirq.ops.I`, `cirq.ops.ControlledGate`, and `cirq.ops.MatrixGate` ?

Our transpiler operates like a hub and spokes model with Cirq at the center, so getting as much coverage testing as we can for all the most common Cirq conversions is super important.

Reply:
Thank you for your feedback on my pull request! I have made the necessary changes to enhance the coverage testing for the CIRQ conversions, as you suggested. In total there is around 62 gates being tested right now. I have renamed the testing files to avoid a conflict with similar pre-existing files with the same name like the: `qbraid/transpiler/cirq_braket/tests/test_from_braket.py`.

Please let me know if there are any further modifications or improvements you would like to see.